networks/p2p/discover/node.go

```diff
@@ -208,8 +216,19 @@ func parseComplete(rawurl string) (*Node, error) {
 	if tcpPort, err = strconv.ParseUint(u.Port(), 10, 16); err != nil {
 		return nil, errors.New("invalid port")
 	}
-	udpPort = tcpPort
+	// Extract subport from query
 	qv := u.Query()
+	if qv.Get("subport") != "" {
+		for _, subport := range qv["subport"] {
+			if p, err := strconv.ParseUint(subport, 10, 16); err != nil {
+				logger.Warn("skipping invalid subport in query", "id", id, "ip", ip, "subport", p)
```

Comment:
Shouldn't it return an error? The user put `subport` in query and may put wrong port number by mistake. What do you think? Also, the failing udp port parsing returns an error.

Reply:
It is a subport not main port. It could be updated later with a correct one.
I think a warning message is fine. It would be cumbersome for a user to fix and run the server again.

Reply:
The subport is used for tx propagation. I think the wrong subport may cause some problems for transaction propagations.

Reply:
Doesn't the propagation happen after handshake? I meant ths subport is updated in handshake process, as I changed in [PR#808](https://github.com/klaytn/klaytn/pull/808).

Reply:
Right. So, the subport can be updated automatically. But, I think it still does not make sense a bit since it accepts wrong scheme.